AndroidBound/src/main/java/solutions/alterego/androidbound/android/viewresolvers/ViewResolver.java

```diff
@@ -111,13 +117,17 @@ public View createView(String name, Context context, AttributeSet attrs) {
         }
 
         if (mappings.containsKey(result)) {
+            logger.debug(getClass().getSimpleName() + " mapping " + name + " to " + mappings.get(result));
             return mappings.get(result);
         }
 
         try {
-            logger.debug("Resolving " + name + " with " + result);
+            logger.debug(getClass().getSimpleName() + " resolving " + name + " with " + result);
```

Comment:
`getClass().getSimpleName()` is quite expensive. You could extract it a a constant

Reply:
I wanted it to print the name of the class, also those who extend it (like `SupportViewResolver`). Generally the idea is not to do logging in general when viewbinding unless debugging because it does slow things down.

Reply:
you can still assign it to a constant.